.github/workflows/update-nimbus-experiments.yml

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+name: "Update Nimbus Experiments"
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+  workflow_dispatch: {}
+
+jobs:
+  update-nimbus-experiments:
+    name: "Update Nimbus Experiments"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout Main Branch"
+        uses: actions/checkout@v2
+        with:
+          path: fenix
+          ref: main
+      - name: "Update Experiments JSON"
+        id: update-experiments-json
+        uses: mozilla-mobile/update-initial-experiments@v1
+        with:
+          repo-path: fenix
+          output-path: app/src/main/res/raw/initial_experiments.json
+          experimenter-url: https://experimenter.services.mozilla.com/api/v6/experiments/?is_first_run=true
+          app-name: fenix
```

Comment:
Drive-by: what's the difference between app-name and repo-name?

Reply:
The action's docs have more info: https://github.com/mozilla-mobile/update-initial-experiments

TL;DR: app-name is used to filter experiments downloaded from experimenter, repo-path is the relative or full path to the checked out repository within the workflow.